runelite-mixins/src/main/java/net/runelite/mixins/VerboseInventoryMixin.java

```diff
@@ -0,0 +1,35 @@
+package net.runelite.mixins;
+
+import java.awt.Container;
+import java.awt.Dimension;
+import net.runelite.api.Constants;
+import net.runelite.api.mixins.Copy;
+import net.runelite.api.mixins.Inject;
+import net.runelite.api.mixins.Mixin;
+import net.runelite.api.mixins.Replace;
+import net.runelite.rs.api.RSClient;
+
+@Mixin(RSClient.class)
+public abstract class VerboseInventoryMixin implements RSClient
```

Comment:
VerboseInventoryQuantitiesMixin? It only makes the quantities more verbose, not the inventory in it's entirety